website/content/docs/templates/legacy_json_templates/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+description: >
+  The legacy JSON templates are JSON files that configure the various components
+  of Packer in order to create one or more machine images. Templates are
+  portable, static, and readable and writable by both humans and computers.
+  This has the added benefit of being able to not only create and modify
+  templates by hand, but also write scripts to dynamically create or modify
+  templates.
+page_title: Legacy JSON Templates
+sidebar_title: Legacy JSON Templates
+---
+
+# Legacy JSON Templates
```

Comment:
```suggestion
# JSON Templates
```

I think having a blurb about JSON templates being a legacy configuration in the partial when ready could be just as effective as is putting Legacy in the section heading.